posts/2022/02/making-gpus-accessible-to-pydata-ecosystem-via-array-api.md

```diff
@@ -0,0 +1,481 @@
+<!--
+.. title: Making GPUs accessible to PyData Ecosystem via Array API
+.. slug: making-gpus-accessible-to-pydata-ecosystem-via-array-api
+.. date: 2022-02-01 21:07:02 UTC-06:00
+.. author: Amit Kumar
+.. tags: GPU, SciPy, NumPy, CuPy, scikit-learn, scikit-image, Array API
+.. category: 
+.. link: 
+.. description: 
+.. type: text
+.. previewimage: /images/2022/02/array_api_workflow.svg
+-->
+
+
+
+GPUs have become an essential part of the scientific computing stack and with
+the advancement in the technology around GPUs and the ease of accessing a GPU
+on cloud or on-prem, it is in the best interest of the PyData community to spend
+time and effort to make GPUs accessible for the users of PyData libraries. A
+typical user in the PyData ecosystem is quite familiar with the APIs of libraries
+like scipy, scikit-learn, scikit-image and at the moment all these
+libraries only support single core operations on CPU. In this blog post will
```

Comment:
I'll rephrase this to reflect that there are parts (`linalg`, `multiprocessing`, scikit-learn's OpenMP usage) are multi-threaded. We can add a footnote to explain that better perhaps.

Reply:
By footnote you mean at the bottom of the post?

Reply:
Yes, but I changed my mind on that one - just adding a few details in brackets.